standard/links_stream.lua

```diff
@@ -34,30 +35,32 @@ StreamLinks.countdownPlatformNames = {
 	'loco',
 	'mildom',
 	'nimo',
-	'stream',
+	TLNET_STREAM,
 	'tl',
 	'trovo',
 	'twitch',
-	'twitch2',
 	'youtube',
 }
-
---[[
-Lookup table of allowed inputs that use a plattform with a different name
-]]
-StreamLinks.streamPlatformLookupNames = {
-	twitch2 = 'twitch',
+local PLATFORM_TO_ICON = {
+	cc163 = 'cc',
+	huomao = 'huomaotv',
+	huya = 'huyatv',
+	himo = 'nimotv',
```

Comment:
Could we rename the actual icons instead?

Reply:
we could add them additional and phase out the old ones
mind we do the same mapping in the infobox icons

Reply:
icon renaming would have impact on e.g. infobox
could rename the inputs instead